fcs/global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
```

Comment:
This may not be necessary because the global.json at the root of this repo is also 3.0.100.

@brettfo is that accurate?

Reply:
So I didn't add this for the dotnet cli, per se, but for FAKE itself. Fake's dotnet cli support will probe for current directory to find the global.json when you specify 'I want to use the version from the global.json'.  A few ways forward:

* work with FAKE to implement the global.json directory-walking algorithm (docs pls?)
* make this a version string in the build.fsx and deal with hand-updating it
* copy the global.json from the repo root to this directory? Note that the root global.json doesn't actually specify an sdk an so FAKE doesn't recognize it as valid for this purpose.

Reply:
EDIT: most of that^ is wrong. FAKe will actually probe upwards, per spec, until it finds a global.json. The problem I had was that there's not actually an `sdk.version` element in the root global.json. If I add one everything works as expected. Is it cool to add 3.0.100 as the sdk to that file?

Reply:
@brettfo could comment on if that's allowable. Since it's used by our cross-repo build system it might not be okay. but you never know.

Reply:
I like having this separate `global.json` here because it helps prevent cross pollution between the FCS build and the root Arcade build, because they're completely separate beasts.